freud/order.pyx

```diff
@@ -607,351 +605,180 @@ cdef class Steinhardt(PairCompute):
             return None
 
 
-cdef class SolLiq(Compute):
-    R"""Uses dot products of :math:`Q_{lm}` between particles for clustering.
+cdef class SolidLiquid(PairCompute):
+    R"""Identifies solid-like clusters using dot products of :math:`Q_{lm}`.
+
+    The solid-liquid order parameter [Frenkel1995]_ uses a Steinhardt-like
+    approach to identify solid-like particles. First, a bond parameter
+    :math:`Q_l(i, j)` is computed for each neighbor bond.
+
+    If :code:`normalize_Q` is true (default), the bond parameter is given by
+    :math:`Q_l(i, j) = \frac{\sum_{m=-l}^{l} \text{Re}~Q_{lm}(i) Q_{lm}^*(j)}
+    {\sqrt{\sum_{m=-l}^{l} \lvert Q_{lm}(i) \rvert^2}
+    \sqrt{\sum_{m=-l}^{l} \lvert Q_{lm}(j) \rvert^2}}`
 
+    If :code:`normalize_Q` is false, then the denominator of the above
+    expression is left out.
+
+    Next, the bonds are filtered to keep only "solid-like" bonds with
+    :math:`Q_l(i, j)` above a cutoff value :math:`Q_{threshold}`.
+
+    If a particle has more than :math:`S_{threshold}` solid-like bonds, then
+    the particle is considered solid-like. Finally, solid-like particles are
+    clustered.
+
+    .. moduleauthor:: Bradley Dice <bdice@bradleydice.com>
     .. moduleauthor:: Richmond Newman <newmanrs@umich.edu>
```

Comment:
I don't know if you want to remove these here or in a future PR (in case it conflicts with the changes you made in #461), but it should be moved in one or the other.

Reply:
I'll fix it in whichever PR is merged second.

Reply:
Fixing here.